src/Spatial/Units/Angle.cs

```diff
@@ -68,6 +83,15 @@ private Angle(double radians)
         /// </summary>
         public double Degrees => this.Radians * RadToDeg;
 
+        /// <summary>
+        /// Explicit conversion from a numeric angle in radians to an Angle object
+        /// </summary>
+        /// <param name="radians">The angle in radians</param>
+        public static explicit operator Angle(double radians)
```

Comment:
Not sure about this.

Reply:
if you have performed a calulcation involving angles it is odd to then need to go Angle.FromRadians(x) to turn it into an angle.  The double is an angle, logically I don't want to "construct" an angle, I have one, I just want the compiler to recognize it as what I say it is and offer me additional functionality that is available to the thing called "Angle"

Reply:
I'm not a fan either. There are multiple interpretations of what an explicit conversion from `double` to an `Angle` could mean (radians? degrees?), and promoting a single one is a great way to introduce unclear code.

Reply:
The trig functions in the .net framework all use Radians.  In fact anywhere in the framework that you need to do something with an angle requires a single or double representing the angle in radians.  I would say that the boat promoting a single meaning has long since sailed in the .net world...